### DISCUSSION TITLE
How to pass first task's output as second task's input?

### DISCUSSION THREAD

**Discussion by sunphy** [apache/dolphinscheduler#9339] [Q&A]
Dolphin is a great project!
I tried dolphin to transfer data from SQL Server to PostgreSQL, which I think it is a very simple scenario. But after digging for a while, I have trouble with how the first task's query resultset can be passed to the next task which is an insert SQL. 
I checked the online document, but get no luck. Thank you, guys.

**Comment by Tandoy** [comment]
Dolphin currently only supports the transfer of task parameters
https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/parameter/context.html

BTW, it seems that this should not be the work of dolphins. Data transmission can refer to the following items:
https://github.com/apache/incubator-seatunnel
https://github.com/alibaba/DataX
https://github.com/wgzhao/Addax

**Comment by sunphy** [comment]
OK. Maybe the philosophy of dolphin is different from what my scenario needed. I think it would be much better if dolphin can handle simple transformation needs, though. 
Thank you all the same, great project.
